.changeset/plenty-files-appear.md

```diff
@@ -0,0 +1,7 @@
+---
+"@frameless/pdc-dashboard": patch
+"@frameless/vth-dashboard": patch
+"@frameless/strapi-plugin-open-forms-embed": patch
+---
+
+Maak gebruik van de nieuwe open-formulieren API versie(2.6.7). Deze wijzigingen lossen de vertragingen op in het veld voor open formulieren in Strapi."
```

Comment:
De technische achtergrond hoeft niet het hoofd-onderwerp te zijn.

Bijvoorbeeld:
Als je in Strapi een link maakt naar een formulier, dan hoef je niet meer zo lang te wachten tot de lijst met alle formulieren is geladen.